src/store/modules/aeternity.js

```diff
@@ -13,6 +13,7 @@ import { BigNumber } from 'bignumber.js';
 import { IS_MOBILE_DEVICE } from '../../utils';
 
 export default {
+  namespaced: true,
```

Comment:
Agree, we should to use it because it's more useful than keep modules without specific namespace.